data_model/src/visit.rs

```diff
@@ -84,6 +82,7 @@ pub trait Visit {
         visit_find_trigger_by_id(&FindTriggerById),
         visit_find_trigger_key_value_by_id_and_key(&FindTriggerKeyValueByIdAndKey),
         visit_find_triggers_by_domain_id(&FindTriggersByDomainId),
+        visit_find_executor_data_model(&FindExecutorDataModel),
```

Comment:
I think end users would appreciate more if we had `FindPermissionTokens` and `FindParameters`. But we can discuss that in a separate issue

Reply:
Created #4618